use new installment criteria in products query (#242)
vtex-apps/product-summary: docs/ProductSummaryList.md

| `hideUnavailableItems` | `Boolean`                              | Hides items that are unavailable.                                                                                                                                                                                                         | `false`       |
| `maxItems` | `Number`                              | Maximum items to be fetched.                                                                                                                                                                                                         | `10`       |
| `skusFilter`               | `SkusFilterEnum`                 | Control SKUs returned for each product in the query. The less SKUs needed to be returned, the more performant your shelf query will be.                                                                                                    | `"ALL_AVAILABLE"` |

> | `installmentCriteria`               | `InstallmentCriteriaEnum`                 | ControlControl what pricee to be shown when price has different installments options.                                                                                                    | `"MAX_WITHOUT_INTEREST"` |

Reviewer: ```suggestion
| `installmentCriteria`               | `InstallmentCriteriaEnum`                 | ControlControl what price to be shown when price has different installments options.                                                                                                    | `"MAX_WITHOUT_INTEREST"` |
```

For `SkusFilterEnum`:

| Name | Value | Description |
| ---- | ----- | ----------- |
| First Available | `FIRST_AVAILABLE` | Most performant, ideal if you do not have a SKU selector in your shelf. Will return only the first available SKU for that product in your shelf query. |
| All Available | `ALL_AVAILABLE` | A bit better performace, will only return SKUs that are available, ideal if you have a SKU selector but still want a better performance. |
| All | `ALL` | Returns all SKUs related to that product, least performant option. |

For `InstallmentCriteriaEnum`:

| Name | Value | Description |
| ---- | ----- | ----------- |

> | Maximum without interest | `MAX_WITHOUT_INTEREST` | Will display price with maximum value before interest. |

Reviewer: ```suggestion
| Maximum without interest | `MAX_WITHOUT_INTEREST` | Will display the maximum installment option with no interest. |
```

> | Maximum with interest | `MAX_WITH_INTEREST` | Will display price with maximum value after interest applied. |

Reviewer: ```suggestion
| Maximum | `MAX_WITH_INTEREST` | Will display the maximum installment option having interest or not. |
```
